components/InviteAdminModal/InviteAdminModal.tsx

```diff
@@ -0,0 +1,223 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import styles from './InviteAdminModal.module.css';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import Icon from "../../assets/Icon";
+
+export default function InviteAdminModal() {
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const [state, setState] = React.useState({
```

Comment:
Since the invite modal could potentially have more admins, we'd need more useState variables, and that can get messy. I'd recommend for you to look at the npm package `react-hook-form`! It will simplify keeping track of all form data. I have an example of it in my auth PR for the sign in page and here's some documentation (https://react-hook-form.com/get-started). If you need any help with this, please let me know!

Reply:
I also think using react hook form will simplify your next spring task of connecting to the backend!